packages/dht/src/connection/webrtc/BrowserWebrtcConnection.ts

```diff
@@ -166,17 +223,26 @@ export class NodeWebrtcConnection extends EventEmitter<Events> implements IWebrt
     }
 
     public send(data: Uint8Array): void {
+        this.messageQueue.push(data)
+        this.messageQueue.sendBuffer()
+    }
+
+    private doSend(data: Uint8Array): void {
         if (this.lastState === 'connected') {
-            this.dataChannel?.send(data as Buffer)
+            if (this.dataChannel!.bufferedAmount > 1 * 1000 * 1000) {
+                this.messageQueue.pause()
```

Comment:
This looks a bit dangerous, doSend() is not an arrow function, so it is not guarantedd that "this" points to the right object. Please make doSend() an arrow function.

Reply:
In line 96 the doSend function is bound to the message queue using an arrow function. No need to make the class method itself in to an arrow function